src/dotnet-core-uninstall/Windows/RegistryQuery.cs

```diff
@@ -72,7 +72,7 @@ internal static bool IsNetCoreBundle(string displayName, string displayVersion,
                     (displayName.IndexOf("Visual Studio", StringComparison.OrdinalIgnoreCase) < 0) &&
                     (displayName.IndexOf("VS 2015", StringComparison.OrdinalIgnoreCase) < 0) &&
                     (displayName.IndexOf("Local Feed", StringComparison.OrdinalIgnoreCase) < 0) &&
-                    ((displayName.IndexOf(".NET Core", StringComparison.OrdinalIgnoreCase) >= 0) ||
+                    ((displayName.IndexOf(".NET", StringComparison.OrdinalIgnoreCase) >= 0) ||
```

Comment:
Does this risk pickup on .NET Framework items? I don't think it should since .NET FX is part of Windows, but I have seen products call out other products like "XYZ for .NET 4.6".  Something we can maybe look into later